recirq/qcqmc/trial_wf.py

```diff
@@ -212,6 +213,97 @@ def _json_dict_(self):
         simple_dict["params"] = self.params
         return simple_dict
 
+    @classmethod
+    def build_pp_plus_trial_from_dependencies(
+        cls,
+        params: PerfectPairingPlusTrialWavefunctionParams,
+        *,
+        dependencies: Dict[data.Params, data.Data],
+        do_print: bool = False,
+    ) -> "TrialWavefunctionData":
+        """Builds a TrialWavefunctionData from a TrialWavefunctionParams
+
+        Args:
+            params: The parameters specifying the PP+ trial wavefunction.
+            dependencies: Data dependencies
```

Comment:
Can we add more information to this arg?

Reply:
Done.